### DISCUSSION TITLE
Does this library work with the "Essential" access tier of v2?

### DISCUSSION THREAD

**Discussion by Lonas13** [xdevplatform/search-tweets-python#152] [Q&A]
Newbie here, apologies if this is a poorly phrased question, but I just recently signed up for the basic level of the Twitter v2 API and am struggling to figure out if I can even use this library or if I'm stuck using the requests library in Python. 

Thanks!

**Comment by igorbrigadir** [comment]
Yes! Everything will work, except you're limited to search results from the last 7 days, (the full archive search and requires academic access) everything else should work the same.

**Reply by Lonas13** [reply]
Awesome! Is there any documentation available on how to handle the credentials for the basic version of v2?

EDIT: Nevermind, I figured it out, I misunderstood the documentation, I didn't have the YAML key indented properly. I now realize that the intended use of the YAML file is to allow each endpoint URL to be referenced in it's own yaml_key variable which is passed to the load_credentials function.
